packages/components/src/components/side-note/side-note.tsx

```diff
@@ -4,21 +4,32 @@ import Icon from '../icon/icon';
 import Text from '../text';
 import './side-note.scss';
 
+const type_to_icon_mapper = {
+    warning: 'IcAlertWarning',
+    information: 'IcAlertInfo',
+    announcement: 'IcAlertAnnounce',
+    error: 'IcAlertDanger',
+};
+
 type TProps = {
+    type?: 'warning' | 'information' | 'announcement' | 'error';
```

Comment:
Can we use key of the `type_to_icon_mapper` for this type?

Reply:
As `type` is only used for icon, Isn't it better to have more clear name for it? for example `icon_type`?

Reply:
> Can we use key of the type_to_icon_mapper for this type?

Nice, I will do the same for `<InlineMessage />` 🙇🏻 

> As type is only used for icon, Isn't it better to have more clear name for it? for example icon_type?

It might be used for colors or other things later in the design, I think `type` is more generic.